generated/storage/group.pb.go

```diff
@@ -98,10 +98,11 @@ func (m *Group) Clone() *Group {
 
 // GroupProperties defines the properties of a group. Groups apply to users when
 // their properties match. For instance:
-//   * If GroupProperties has only an auth_provider_id, then that group applies
```

Comment:
I suppose the comments in the source proto files should be updated and pushed as well.

Reply:
Looking at the `grouped-static-checks` tests, I think the styling changes are required in the proto files to ensure the code re-generation does not erase the fixes later.

Reply:
sure we can be on the safe side